### PR TITLE
fix: ignore vuln with no upgrade and bump snyk-delta and snyk config

### PR DESCRIPTION
**Pull request by mathild3r**
- [ ] Tests written and linted [ℹ︎](https://github.com/snyk/general/wiki/Tests)
- [ ] Documentation written [ℹ︎](https://github.com/snyk/general/wiki/Documentation)
- [ ] Commit history is tidy [ℹ︎](https://github.com/snyk/general/wiki/Git)

### What this does

bump lib and set ignore vuln for 30 days if no patch available

### Notes for the reviewer

_Instructions on how to run this locally, background context, what to review, questions…_

### More information

- [Link to documentation](https://github.com/snyk/snyk-prevent-gh-commit-status/wiki/)

### Screenshots

_Visuals that may help the reviewer_

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -39,8 +39,8 @@
     "axios": "^0.27.2",
     "debug": "^4.1.1",
     "lodash": "^4.17.21",
-    "snyk-delta": "^1.8.0",
     "snyk-config": "^4.0.0",
+    "snyk-delta": "^1.5.4",
```
Comment (lili2311) [inline package.json]: this is a downgrade compare to `^1.8.0`, what should be the min version now?

Comment (mathild3r) [reply package.json]: oh sorry

Comment (mathild3r) [reply package.json]: that one, sorry reverting this

**Review by lili2311 — APPROVED** [review]

**Pull request merged**